docs/operator/flux-kustomize.md

```diff
@@ -191,4 +191,65 @@ spec:
           kind: Kustomization
 ```
 
+### Cluster sync GitRepository verification
+
+```yaml
+apiVersion: fluxcd.controlplane.io/v1
+kind: FluxInstance
+spec:
+  kustomize:
+    patches:
+      - patch: |
+          - op: replace
+            path: /spec/verify
+            value:
+              mode: HEAD
+              secretRef:
+                name: pgp-public-keys
+        target:
+          kind: OCIRepository
+          name: flux-system
+```
+
+### Cluster sync OCIRepository verification
```

Comment:
Let's keep only keyless, people can read the docs and see all the verification options.